README.md

```diff
@@ -0,0 +1,198 @@
+# Hello-World-Microverse-Week1
+
+<a name="readme-top"></a>
+
+
+
+
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World Project] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences. This is my first project in Microverse
```

Comment:
- [ ] I think it would be better in the description to tell what your project is about because just by reading we should have a clear picture of your project. Please consider using the following description: :point_down: 
`This project is about setting up all required tools and standards (e.g. Linters) in a simple "Hello Microverse" project.` 👍️

Reply:
Dear @ridwanediallo, I changed the description as you suggested. Is it good enough now to get approval? Thanks